README.md

```diff
@@ -45,6 +45,11 @@ bundle install
 yarn install
 ```
 
+This Project uses teh rbnacl gem, and requires libsodium on mac:
```

Comment:
what is rbnacl? what is libsodium? Why?

Reply:
the rbnacl gem, which a Ruby wrapper for the libsodium library. Libsodium is a popular cryptographic library that provides a wide range of cryptographic primitives